spine-js/spine.js

```diff
@@ -196,6 +198,8 @@ spine.Slot.prototype = {
 		this.b = data.b;
 		this.a = data.a;
 
+		this.attachmentVertices = [];
+
```

Comment:
This is done in `setAttachment` (below), which I believe should always be called. The length is set rather than allocating a new array.